src/resources/UsageAnalytics/Read/DataHealth/DataHealthInterfaces.ts

```diff
@@ -310,3 +310,80 @@ export interface DataHealthGetTrackingIdsResponse {
      */
     values: DataHealthFacetValue[];
 }
+
+export interface DataHealthGetEventProblemsParams extends OrganizationParamParts, TimeRangeParamParts, Paginated {
+    /**
+     * A set of event types for which the events will be returned.
+     * If not specified, events are not filtered on event types.
+     */
+    eventType?: string[];
```

Comment:
Should the type be more specific than string, or are there just too many possible values for that?

In the latter case, could the docs point to the analytics schema repository as a reference?

Reply:
I don't think it should be more specific, there should only be one source of truth and the event types are very well documented on docs.coveo.com

Reply:
I could add https://docs.coveo.com/en/2949/analyze-usage-data/events to the comment, but I don't know if we have a policy on url link directly in the comments.

Reply:
I think a link to the docs would be nice, yes